src/__tests__/helpers/auth-test-helpers.ts

```
@@ -0,0 +1,223 @@
+  
+  return {
+    mockCallback,
+    triggerAuthEvent,
+    triggerSignIn: (session: Session) => triggerAuthEvent('SIGNED_IN', session),
+    triggerSignOut: () => triggerAuthEvent('SIGNED_OUT', null),
+    triggerTokenRefreshed: (session: Session) => triggerAuthEvent('TOKEN_REFRESHED', session),
+    triggerPasswordRecovery: (session: Session) => triggerAuthEvent('PASSWORD_RECOVERY', session)
+  }
+}
+
+export function mockAuthContextValue(overrides: any = {}) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)